boot/boot.js

```diff
@@ -2022,7 +2017,7 @@ $tw.boot.startup = function(options, callback) {
 	$tw.boot.executedStartupModules = Object.create(null);
 	$tw.boot.disabledStartupModules = $tw.boot.disabledStartupModules || [];
 	// Repeatedly execute the next eligible task
-	$tw.boot.executeNextStartupTask(callback);
+	$tw.boot.executeNextStartupTask(options.callback);
```

Comment:
Is this null safe? `$tw.boot.executeNextStartupTask(undefined)`? What happens when `options.callback` is not defined or is not a function?